### PR TITLE
Start the shell with the correct `cwd` on Windows

### LINKED ISSUE #1011: Starting CWD on Windows is Always Hyper.exe's Folder
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 build 14393
- **Hyper.app version**: 0.8.3 (installer from AppVeyor artifacts)
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/zbuttram/76b649fc07aaba5602dc8f3186b041b2
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Installed from AppVeyor artifacts https://ci.appveyor.com/project/appveyor-zeit/hyper/build/1.0.121/artifacts

## Issue
Seems like whenever Hyper is started on Windows, the starting CWD is wherever Hyper.exe is run from (as opposed to the user's home or the system root):

image

### PR DESCRIPTION
**Pull request by zbuttram**
This fixes #1011, should be ready to merge.

### CODE REVIEW

**Comment by matheuss** [comment]
Amazing @zbuttram, thanks for your time on this! I think you can switch it to just `process.env.HOME || process.env.HOMEPATH` 😄

File: app/index.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       }
     });
 
-    rpc.on('new', ({rows = 40, cols = 100, cwd = process.env.HOME, splitDirection}) => {
+    rpc.on('new', ({rows = 40, cols = 100, cwd = process.platform === 'win32' ? process.env.HOMEPATH : process.env.HOME, splitDirection}) => {
```
Comment (matheuss) [inline app/index.js]: I think you can switch it to just `process.env.HOME || process.env.HOMEPATH` 😄

Comment (zbuttram) [inline app/index.js]: Oh duh. Thanks, I'll fix it up right now.

**Comment by zbuttram** [comment]
Requested change is in now, still works and much clearer now. Thanks for catching that @matheuss, sometimes I get too ternary happy. ;)

**Comment by matheuss** [comment]
Thank you @zbuttram 🙌

**Comment by sindresorhus** [comment]
This should use `require('os').homedir()`, which is much more dependable than environment variables.

**Pull request merged**